### PR TITLE
https://issues.redhat.com/browse/ACM-12486 Edit what's new

### PR DESCRIPTION
**Pull request by oafischer**
No QE confirmation for Submariner, removed networking

### CODE REVIEW

File: release_notes/whats_new.adoc [hunk 1]
```diff
@@ -81,31 +73,26 @@ See link:../observability/observe_environments_intro.adoc#observing-environments
...
 
-* Use the `recreateOption` field for your configuration policies to delete and recreate an object. See, link:../governance/config_policy_ctrl.adoc#configuration-policy-yaml-table[Configuration policy YAML table].
+* Use the `recreateOption` field for your configuration policies to delete and recreate an object. See link:../governance/config_policy_ctrl.adoc#configuration-policy-yaml-table[Configuration policy YAML table].
 
 * You can now use the `InStatus` option to store the detected difference of the `ConfigurationPolicy` resource. See link:../governance/config_policy_ctrl.adoc#configuration-policy-yaml-table[Configuration policy YAML table].
 
-* You must now use the new parameter, `upgradeApproval` in your `OperatorPolicy` resource to automate version upgrade approvals for your cluster. See, link:../governance/policy_operator.adoc#policy-operator-yaml-table[Operator policy YAML table].
+* Use the new `upgradeApproval` parameter in your `OperatorPolicy` resource to automate version upgrade approvals for your cluster. See, link:../governance/policy_operator.adoc#policy-operator-yaml-table[Operator policy YAML table].
```
Comment (cadawson) [inline release_notes/whats_new.adoc]: Remove comma after See.

File: release_notes/whats_new.adoc [hunk 2]
```diff
@@ -81,31 +73,26 @@ See link:../observability/observe_environments_intro.adoc#observing-environments
...
 
-* Use the new optional parameter, `spec.complianceConfig`, to define the compliance behavior for the operator policy. See, link:../governance/policy_operator.adoc#policy-operator-yaml-table[Operator policy YAML table].
+* Use the new optional `spec.complianceConfig` parameter to define the compliance behavior for the operator policy. See, link:../governance/policy_operator.adoc#policy-operator-yaml-table[Operator policy YAML table].
```
Comment (cadawson) [inline release_notes/whats_new.adoc]: Remove comma after See.

**Review by cadawson — CHANGES_REQUESTED** [review]
Looks good. A couple of nits. No need to see it again. Thanks!

**Review by dockerymick — APPROVED** [review]
Looks good! Thanks Oliver!

**Pull request merged**